.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,38 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.154.0/containers/rust
+{
+	"name": "Rust",
+	"build": {
+		"dockerfile": "Dockerfile"
+	},
+	"runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
```

Comment:
You shouldn't need CAP_SYS_PTRACE with seccomp=unconfined, but maybe it allows you to use an unprivileged lldb to debug an nsncd running as root?

Reply:
This is the default rust config vscode gave me. I didn't add these, but I can remove them if you think we should.

Reply:
lol, okay, leave it and I'll file a bug report against vscode :)

Reply:
https://github.com/microsoft/vscode-remote-try-rust/blob/master/.devcontainer/devcontainer.json#L8